packages/sms/src/rest/v1/enums.ts

```diff
@@ -1 +1 @@
-export type { GetDeliveryReportByBatchIdTypeEnum } from './delivery-reports';
+export {};
```

Comment:
May be this file could be removed

Reply:
We may have some shared enums at some point and this file is there for that. It doesn't hurt to leave it empty for the moment.